src/lib/sidenav/drawer.ts

```diff
@@ -467,29 +520,33 @@ export class MdDrawerContainer implements AfterContentInit {
   }
 
   /**
-   * Return the width of the drawer, if it's in the proper mode and opened.
-   * This may relayout the view, so do not call this often.
-   * @param drawer
-   * @param mode
+   * Recalculates and updates the inline styles for the content. Note that this should be used
+   * sparingly, because it causes a reflow.
    */
-  private _getDrawerEffectiveWidth(drawer: MdDrawer, mode: string): number {
-    return (this._isDrawerOpen(drawer) && drawer.mode == mode) ? drawer._width : 0;
-  }
+  private _updateContentMargins() {
+    let left = 0;
+    let right = 0;
+
```

Comment:
Add comment that describes which style is applied where based on the mode + position?